tools/build/Kconfig.riot

```diff
@@ -9,5 +9,6 @@ config RIOT
     select FEATURE_WORKER_THREADS
     select FEATURE_NETWORK
     select FEATURE_COAP
+    select FEATURE_OIC
```

Comment:
This will require the FEATURE_IPV4 or something like that afterwards, but it's OK to get in nevertheless.